applications/ShapeOptimizationApplication/tests/algorithm_bead_optimization_test/analysis_parameters.json

```diff
@@ -30,9 +30,7 @@
       "residual_absolute_tolerance"        : 1e-9,
       "max_iteration"                      : 10,
       "linear_solver_settings"             : {
-          "solver_type" : "eigen_pardiso_lu",
-          "scaling"     : false,
-          "verbosity"   : 0
+          "solver_type" : "eigen_sparse_lu"
```

Comment:
:+1: 
Does it have a skiptest if EigenSolversApp is not available?

=> I think @dbaumgaertner told me that for this case also the SuperLU would work, but you better check with him